src/Atc.Azure.Messaging/EventHub/EventHubCredentialsPublisherFactory.cs

```diff
@@ -0,0 +1,36 @@
+using Atc.Azure.Options.Authorization;
+using Atc.Azure.Options.Environment;
+using Atc.Azure.Options.EventHub;
+using Atc.Azure.Options.Providers;
+using Azure.Identity;
+using Azure.Messaging.EventHubs.Producer;
+
+namespace Atc.Azure.Messaging.EventHub;
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+
+public class EventHubCredentialsPublisherFactory : IEventHubPublisherFactory
```

Comment:
Since this is an open source project and that we now have consumers of this project, I suggest we add XML code documentation comments to all public facing types

Just a simple `<summary>` should be enough

```
<summary>
Something something blah blah blah
</summary>
```

Reply:
Done